pages/account/index.vue

```diff
@@ -0,0 +1,350 @@
+            :loading="searchingCities || loadingCepChange"
+            required
+          >
+            <ElOption
+              v-for="city in cities"
+              :key="city"
+              :label="city"
+              :value="city"
+            />
+          </ElSelect>
+        </ElFormItem :show-message="false">
+        <ElFormItem :show-message="false" class="item" label="Bairro" prop="address_neighborhood" required>
+          <ElInput v-model="form.address_neighborhood" placeholder="Bairro" />
+        </ElFormItem :show-message="false" class="item">
+        <ElFormItem :show-message="false" class="item" label="Rua" prop="address_street" required>
+          <ElInput v-model="form.address_street" placeholder="Rua" :disabled="loadingCepChange" />
+        </ElFormItem :show-message="false" class="item">
+        <ElFormItem :show-message="false" class="item" label="Número" prop="address_number" required>
+          <ElInput v-model="form.address_number" placeholder="Número" />
+        </ElFormItem :show-message="false" class="item">
+        <ElFormItem :show-message="false" class="item" label="Complemento" prop="address_complement">
+          <ElInput v-model="form.address_complement" placeholder="Complemento" />
+        </ElFormItem :show-message="false" class="item">
+      </div>
+    </ElForm>
+    <CommonCoolFooter hideToggle>
+      <ElButton
+        class="confirm-button"
+        type="primary"
+        size="large"
+        :disabled="invalidChanges || loading"
+        :loading="loading"
+        @click="handleClick"
+        >Salvar alterações</ElButton
+      >
+    </CommonCoolFooter>
+  </div>
+</template>
+
+<style scoped>
+.confirm-button {
+  height: 100%;
+}
+
+.two-columns {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0.5rem;
+  width: 100%;
+}
+
+p {
+  font-size: 1em;
+  margin: 1em 0;
+  color: var(--black-80);
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.75rem;  
+  width: 100%;
+  padding: 1rem 1rem 1rem 1rem;
+  height: calc(100dvh - var(--subheader-size) - 90px - var(--navbar-size) );
+  overflow-y: auto;
+}
+.main {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  min-height: 100%;
+}
+
+.item {
+  width: 100%;
+  margin-bottom: 0;
+}
+
+.item:deep(.el-form-item__label) {
+  margin-bottom: 6px !important;
+}
+</style>
+
+<script setup lang="ts">
+import { cloneDeep } from "lodash";
+import { useUserStore, type UserUpdate } from "@/stores/user";
+
+definePageMeta({
+  pageTransition: {
+    name: "slide-left-fast-and-furious",
+    mode: "out-in",
+  },
+});
+
+const userStore = useUserStore();
+const userCopy = cloneDeep(userStore.user);
+const addressCopy = userCopy?.addresses?.[0];
+
+const phoneNoCountryCode = userCopy?.phone?.replace("+55", "");
+
+if (!userCopy) {
+  throw new Error("User not found");
```

Comment:
Acredito que seja bom mandar o usuário pro login

Reply:
isso dai na real eh meio impossivel - as telas do nuxt só aparecem quando o cara ta logado. usei isso dai mais pra forçar a tipagem mermo kkkkkkkk

Reply:
fiz a boa aqui - tirei o if e forcei a tipagem a ficar certa